### DISCUSSION TITLE
Python / Multi-line Parameter 매개변수 개수 질문

### DISCUSSION THREAD

**Discussion by yunjjun** [8percent/styleguide#44] [Q&A] [answered]
저희 Python / Multi-line Parameter 를 보면
'한 줄에 두 개 이상의 매개변수를 적지 않음' 이라고 되어 있는데 '이상' 의 표현이 확실한 건지 확인해 주시면 감사합니다!
2개부터는 줄 바꿈을 권장하는 것이 맞지요? 코드 리뷰를 위해 정확하게 확인하려고 합니다.

**Comment by kyungjunleeme** [comment]
저도 그렇게 이해하고 있습니다.

**Comment by cbscsm** [comment, marked as answer]
네 맞습니다.
